rust_src/src/buffers.rs

```diff
@@ -52,6 +52,11 @@ impl LispBufferRef {
         unsafe { (*self.text).gpt_byte }
     }
 
+    #[inline]
+    pub fn gap_position(&self) -> ptrdiff_t {
```

Comment:
Is the reference required here? If not leave it off.

Reply:
I think somebody mentioned, that we should do this for consistency reasons some time ago. @Wilfred ?

Reply:
IMO we should reorganize our impls.

Reply:
I have a PR open to remove refs.#491

Reply:
I removed the reference for this function, so that it doesn't conflict with your PR.